okhttp-tests/src/test/java/okhttp3/CallTest.java

```diff
@@ -243,10 +244,9 @@
         .url(server.url("/"))
         .head()
         .build();
-    executeSynchronously(headRequest)
-        .assertCode(200)
-        .assertHeader("Content-Length", "100")
-        .assertBody("");
+    Response response = client.newCall(headRequest).execute();
+    assertEquals(200, response.code());
+    assertArrayEquals(new byte[0], response.body().bytes());
```

Comment:
Here's the test. The string() method gets away without doing an extra length check.

Reply:
Derp. Blame mobile.